From aa8b64f6e1979ed8d2434a3943e325617103c34d Mon Sep 17 00:00:00 2001
From: Herrie <Github.com@herrie.org>
Date: Thu, 4 Jan 2024 10:04:53 +0100
Subject: [PATCH] luna-init: CMakeLists.txt: Remove checks for
 WEBOS_TARGET_DISTRO

Since we don't need these on LuneOS.

Signed-off-by: Herman van Hazendonk <github.com@herrie.org>
---
Upstream-Status: Inappropriate [LuneOS specific]

 CMakeLists.txt | 21 +++------------------
 1 file changed, 3 insertions(+), 18 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 5690eb4..cebc513 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -75,24 +75,9 @@ install(DIRECTORY files/conf/fonts DESTINATION ${WEBOS_INSTALL_DATADIR}
         PATTERN ".txt" PERMISSIONS OWNER_READ OWNER_WRITE OWNER_EXECUTE
 )
 
-if(${WEBOS_TARGET_DISTRO} STREQUAL "signage")
-    install(FILES files/conf/signage/defaultPreferences.txt DESTINATION ${WEBOS_INSTALL_WEBOS_SYSCONFDIR}
+install(FILES files/conf/defaultPreferences.txt DESTINATION ${WEBOS_INSTALL_WEBOS_SYSCONFDIR}
         PERMISSIONS OWNER_READ OWNER_WRITE GROUP_READ WORLD_READ)
-elseif(${WEBOS_TARGET_DISTRO} STREQUAL "starfish")
-    install(FILES files/conf/starfish/defaultPreferences.txt DESTINATION ${WEBOS_INSTALL_WEBOS_SYSCONFDIR}
+install(FILES files/conf/cust-preferences.txt DESTINATION ${WEBOS_INSTALL_SYSMGR_DATADIR}/customization
         PERMISSIONS OWNER_READ OWNER_WRITE GROUP_READ WORLD_READ)
-elseif(${WEBOS_TARGET_DISTRO} STREQUAL "webos-pro")
-    install(FILES files/conf/webos/defaultPreferences.txt DESTINATION ${WEBOS_INSTALL_WEBOS_SYSCONFDIR}
+install(FILES files/conf/CustomerCareNumber.txt DESTINATION ${WEBOS_INSTALL_WEBOS_SYSCONFDIR}
         PERMISSIONS OWNER_READ OWNER_WRITE GROUP_READ WORLD_READ)
-elseif(${WEBOS_TARGET_DISTRO} STREQUAL "apollo")
-    install(FILES files/conf/apollo/defaultPreferences.txt DESTINATION ${WEBOS_INSTALL_WEBOS_SYSCONFDIR}
-        PERMISSIONS OWNER_READ OWNER_WRITE GROUP_READ WORLD_READ)
-else()
-    install(FILES files/conf/defaultPreferences.txt DESTINATION ${WEBOS_INSTALL_WEBOS_SYSCONFDIR}
-        PERMISSIONS OWNER_READ OWNER_WRITE GROUP_READ WORLD_READ)
-    install(FILES files/conf/cust-preferences.txt DESTINATION ${WEBOS_INSTALL_SYSMGR_DATADIR}/customization
-        PERMISSIONS OWNER_READ OWNER_WRITE GROUP_READ WORLD_READ)
-    install(FILES files/conf/CustomerCareNumber.txt DESTINATION ${WEBOS_INSTALL_WEBOS_SYSCONFDIR}
-        PERMISSIONS OWNER_READ OWNER_WRITE GROUP_READ WORLD_READ)
-
-endif()
\ No newline at end of file
